packages/server/src/async/consumers/consumer.js

```diff
@@ -0,0 +1,30 @@
+const assert = require('assert');
+const _ = require('lodash')
+
+const data = require('../../data');
+
+const { TEMPLATE_NAME } = require('../../models/enums/template-name');
+const { NOTIFICATION_TYPE } = require('../../models/enums/notification-type');
+const { EmailEnvelope } = require('../../models/classes/notifications');
+
+class Consumer {
```

Comment:
Consumer seems kind of vague. Maybe “NotificationConsumer” or similar?

Reply:
Here "consumer" class is parent class of all another "consumers", for instance "EmailsConsumer", "PushConsumer" and another ones.

Reply:
Can you write these comments in the code, instead of in replies? That makes sense to me, but I will forget.

Reply:
Sure.